api/src/pcapi/routes/backoffice/accounts/update_request_blueprint.py

```diff
@@ -505,15 +505,20 @@ def get_ask_for_correction_form(ds_application_id: int) -> utils.BackofficeRespo
         raise NotFound()
 
     form = empty_forms.EmptyForm()
+    correction_reason = request.args["correction_reason"]
 
     return render_template(
         "components/dynamic/modal_form.html",
         target_id=f"#request-row-{ds_application_id}",
         form=form,
-        dst=url_for("backoffice_web.account_update.ask_for_correction", ds_application_id=ds_application_id),
-        div_id=f"ask-for-correction-{ds_application_id}",
+        dst=url_for(
+            "backoffice_web.account_update.ask_for_correction",
+            ds_application_id=ds_application_id,
+            correction_reason=correction_reason,
+        ),
+        div_id=f"ask-for-correction-{correction_reason}-{ds_application_id}",
         title="Demander une correction",
-        information="Ce message sera envoyé au jeune: <br>" + users_ds.CORRECTION_MESSAGE,
+        information="Ce message sera envoyé au jeune: <br>" + users_ds.CORRECTION_MESSAGE[correction_reason],
```

Comment:
Suggestion d'amélioration par rapport à l'existant : reproduire les retours à la ligne.
Détail :
```suggestion
        information=Markup("Ce message sera envoyé au jeune : <br><br>") + nl2br(users_ds.CORRECTION_MESSAGE[correction_reason]),
```